include/freerdp/server/shadow.h

```diff
@@ -99,7 +99,7 @@ extern "C"
 	typedef BOOL (*pfnShadowChannelAudinServerReceiveSamples)(rdpShadowSubsystem* subsystem,
 	                                                          rdpShadowClient* client,
 	                                                          const AUDIO_FORMAT* format,
-	                                                          wStream* buf, size_t nframes);
```

Comment:
Any reason for not keeping the `wStream*` here?

Reply:
Uses now a `wStream *` since the last push.